client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -117,6 +117,7 @@ export function WPOrderReviewLineItems( {
 			{ couponLineItem && (
 				<WPOrderReviewListItem key={ couponLineItem.id }>
 					<NonProductLineItem
+						responseCart={ responseCart }
```

Comment:
Passing cart data to _NonProductLineItem_ so that it can tell if there is a partner coupon, since it does not have product information.